src/PrestaShopBundle/Controller/Admin/Improve/Design/PositionsController.php

```diff
@@ -83,7 +83,7 @@ public function indexAction(Request $request)
 
         $modules = [];
         foreach ($installedModules as $installedModule) {
-            if ($module = $moduleAdapter->getInstanceById($installedModule['id_module'])) {
+            if ($installedModule['active'] && $module = $moduleAdapter->getInstanceById($installedModule['id_module'])) {
                 // We want to be able to sort modules by display name
```

Comment:
I hesitated here, I could have:

- modified the `getModulesInstalled` so that it gets only enabled module... but I can't be sure we want this behavior
- adding a method or a parameter to `getModulesInstalled` 

This way seemed the less invasive, allowing the QA team to having fewer side effect checks to make before release candidate.